### PR TITLE
Added Afrikaans language support

### PR DESCRIPTION
**Pull request by JacquesvanWyk**
Hi I was in need for Afrikaans language support for my Laravel application and came across your repo and decided to add the work I have done for Afrikaans language.

### CODE REVIEW

File: src/af/auth.php [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    | Authentication Language Lines
+    |--------------------------------------------------------------------------
+    |
+    | The following language lines are used during authentication for various
+    | messages that we need to display to the user. You are free to modify
+    | these language lines according to your application's requirements.
+    |
+    */
+
+    'failed' => 'Hierdie verwysings stem nie ooreen met ons rekords nie.',
```
Comment (caouecs) [inline src/af/auth.php:16]: Need 'password' => 'The provided password is incorrect.',

File: src/af/validation.php [hunk 2]
```diff
@@ -0,0 +1,185 @@
...
+    |--------------------------------------------------------------------------
+    |
+    | The following language lines contain the default error messages used by
+    | the validator class. Some of these rules have multiple versions such
+    | as the size rules. Feel free to tweak each of these messages here.
+    |
+    */
+
+    'accepted' => 'Die :attribute moet aanvaar word.',
+    'active_url' => 'Die :attribute is nie \'n geldig URL.',
```
Comment (caouecs) [inline src/af/validation.php:17]: Use " when you have ' in translations

For example here :  'active_url' => "Die :attribute is nie 'n geldig URL.",

**Review by caouecs — CHANGES_REQUESTED** [review]
Hello,
thank you, can you add validation-inline.php and fix files ?

**Comment by caouecs** [comment]
@JacquesvanWyk  ?

**Comment by andrey-helldar** [comment]
@JacquesvanWyk, ping 🙂

**Comment by JacquesvanWyk** [comment]
Hey sorry about this been crazy my side. Going to finish this on weekend and send another pull request.

**Comment by andrey-helldar** [comment]
@JacquesvanWyk, is the situation still crazy?)

**Comment by andrey-helldar** [comment]
@JacquesvanWyk, ping)

**Comment by andrey-helldar** [comment]
@caouecs, I propose to accept PR from @JacquesvanWyk. He translated many keys. Yes, not all keys have been translated. The script will generate a todo.md file with missing keys. In any case, it will be better than using Google Translate. What do you say?

**Comment by JacquesvanWyk** [comment]
Hi
I will have been finishing app that i have used this so was just finishing
that. I will send the updated one in next few days

**Comment by JacquesvanWyk** [comment]
Please review

**Review by JacquesvanWyk — COMMENTED** [review]
Inline Validations done and some other fixes

**Comment by caouecs** [comment]
thank you

**Pull request merged**